flocx_market/tests/unit/db/sqlalchemy/test_offer_api.py

```diff
@@ -8,13 +8,13 @@
 
 now = datetime.datetime.utcnow()
 test_offer_data = dict(
-    marketplace_date_created=now,
     provider_id='2345',
     creator_id='3456',
+    marketplace_date_created=now,
+    status='available',
```

Comment:
This appears to be changing the order of the attributes, but otherwise both versions of the code are the same.  What's the reason for the change here?

Reply:
I changed the order because that is the order that it appears in the OfferModel, so it was just easier to look at, do you think we don't need that @larsks?

Reply:
That's fine; I was just curious. We can keep it.